homeassistant/components/energyzero/sensor.py

```diff
@@ -115,6 +113,51 @@ class EnergyZeroSensorEntityDescription(
         icon="mdi:percent",
         value_fn=lambda data: data.energy_today.pct_of_max_price,
     ),
+    EnergyZeroSensorEntityDescription(
+        key="average_price",
+        name="Average - tomorrow",
```

Comment:
HA has got support for [translating entity names](https://developers.home-assistant.io/docs/internationalization/core#name-of-entities) and all other sensors of `energyzero` are already supporting them.

For consistency can you please make the new sensor translatable too?